docs/body.md

```diff
@@ -748,39 +615,336 @@ when this happens.
+
+*   After deployment is complete, go to the undercloud, SSH into controller and go
+    to `/etc/ceilometer` in ceilometer_agent_notification container to check if
+    `pipeline.yaml` and `event_pipeline.yaml` file is updated:
+
+            ssh heat-admin@<controller ip>
+            sudo su -
+            sudo docker exec --user 0 -it ceilometer_agent_notification /bin/bash
+            # Ensure your configuration had the right publishers
+
+#### Notes:
+
+*   Events exposed by Ceilometer are listed in zOpenStackProcessEventTypes.
+    By default, this is set to the single *compute.instance.create.error* .
+
+*   Proxy warnings in [ZPS-4689](#known-issues) may be displayed during the
+    upgrade to 3.0.0 if you have multiple collectors. These can be safely
+    ignored.
+
+*   The Instance metrics for *Disk IO Rate* are deprecated in OpenStack version
+    Queens and later. Collection for those metrics will be missing.
+    Future OpenStack releases will remove these metrics and graphs completely.
+
+    In the meantime, if you still require these metrics, you can edit the
+    OpenStack Ceilometer configuration /etc/ceilometer/polling.yaml and add the
+    following to the *meters* section:
+
+            - disk.read.bytes
+            - disk.read.requests
+            - disk.write.bytes
+            - disk.write.requests
+
+    - Ensure the resulting YAML syntax is valid restart the ceilometer-polling service
+
+    Finally note that these metrics ARE deprecated and will be removed in
+    future releases of OpenStack itself.
+
+    Note: **There is no known template configuration for this for RHOSP
+          templates at this time and are unsupported.**
+
+*   Make sure you restart ceilometer-notification service whenever you change
+    `pipeline.yaml` or `event_pipeline.yaml`.
+
+*   For Queens and Rocky devices, make sure to restart ceilometer-polling service
+    if you have updated `/etc/ceilometer/polling.yaml`
+
+*   The RabbitMQ-Ceilometer service
+
+    `RabbitMQ-Ceilometer` is typically only needed for legacy (pre 3.0,0 ZenPack) systems.
+
+    Version 2.4.0 of this zenpack introduces a new service, `RabbitMQ-Ceilometer`.
+    This is a dedicated instance of RabbitMQ on each collector which is used solely
+    for integration with Ceilometer, rather than using the standard `RabbitMQ`
+    service that is used by Zenoss itself. This better distributes any load as
+    well as providing better support for distributed collector scenarios where the
+    target OpenStack environment might not have network access to the central
+    Zenoss servers.
+
+*   Zenoss collectors need direct access to the NovaHost devices in order to
+    model them. In some setups like RHOSP's overcloud, where direct
+    access to the overcloud network is not accessible by default, this can be a
+    challenge. Discussion of how to achieve connectivity to overcloud is
+    out of scope.
+
+### Optional Steps
+
+#### VM State Changes
```

Comment:
I would suggest moving this into the "Manual Configuration".. and honestly we can probably simplify the instructions.  I feel like i was a little too verbose here.   :)